db/migrate/20160510102723_create_bank_statement_items.rb

```diff
@@ -0,0 +1,21 @@
+class CreateBankStatementItems < ActiveRecord::Migration
+  def change
+    create_table :bank_statement_items do |t|
+      t.references :bank_statement, null: false, unique: true, index: true
+      t.string     :name,           null: false
+      t.decimal    :debit,          null: false, precision: 19, scale: 4, default: 0.0
+      t.decimal    :credit,         null: false, precision: 19, scale: 4, default: 0.0
+      t.string     :currency,       null: false
+      t.date       :transfered_on,  null: false
+      t.date       :initiated_on
+      t.string     :transaction_id
```

Comment:
[We might want to change the name because an attribute ending with `_id` generates a `bin/rake clean:schema` error as it interprets it as a relation](https://github.com/ekylibre/ekylibre/blob/master/lib/tasks/clean/schema.rake#L44-L68)